ui/lib/core/addon/components/key-value-header.js

```diff
@@ -1,87 +1,69 @@
-import { computed } from '@ember/object';
-import Component from '@ember/component';
+import Component from '@glimmer/component';
 import utils from 'vault/lib/key-utils';
-import layout from '../templates/components/key-value-header';
 import { encodePath } from 'vault/utils/path-encoding-helpers';
 
-export default Component.extend({
-  layout,
-  tagName: 'nav',
-  classNames: 'key-value-header breadcrumb',
-  ariaLabel: 'breadcrumbs',
-  attributeBindings: ['ariaLabel:aria-label', 'aria-hidden'],
+/**
+ * @module KeyValueHeader
+ * KeyValueHeader components show breadcrumbs.
```

Comment:
Thanks for adding the `jsdoc` - does it just show breadcrumbs? Or is it breadcrumbs specifically for the secrets engines?

Reply:
It really only shows breadcrumbs. The hbs template is essentially two links separated by the > icon. Regarding if it's specific for secret-engines. The component is used only in secret engines, however, the styling of the component (e.g. the class) is used outside of secrets engine. example, for the class "key-value-header" it's used in replication.

Reply:
gotcha - thanks!